controllers/apartments_controller.rb

```diff
@@ -0,0 +1,36 @@
+get "/apartments" do
+  @all_apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+get "/apartments/new" do
+  erb :"apartments/new"
+end
+
+get "/apartments/:id" do
+  @apartments = Apartment.all
+  erb :"apartments/id"
+end
+
+put "/apartments/:id" do
+  @address = params[:address]
+  @rent = params[:monthly_rent]
+  @sqft = params[:sqft]
+  @beds = params[:num_beds]
+  @baths = params[:num_baths]
+  Apartment.create(address: @address, monthly_rent: @rent, sqft: @sqft, num_beds: @beds, num_baths: @baths)
```

Comment:
This looks like an update action, so I would recommend finding the specific apartment, then calling update on that instance and passing in the params. Or make this the create route and do: `post '/apartments'`